interface.html

```diff
@@ -527,9 +530,23 @@ <h4 class="panel-title collapsed" role="button" data-toggle="collapse" data-pare
                   <div class="col-sm-4 control-label">
                     <label for="fl-store-releaseNotes">Release Notes</label>
                     <p class="help-block label-helper"><small>When submitting an update to your app, use the Release Notes to tell users what is new in the new version.</small></p>
+                    <p class="help-block label-helper">
+                      <small data-template="fl-store-releaseNotes">
+                        <a>Use template</a>
```

Comment:
This could be why the pointer cursor isn't working. You need a `href` for anchors to be valid.

```suggestion
                        <a href="#">Use template</a>
```